src/lib/multicall/Multicall.ts

```diff
@@ -153,7 +153,12 @@ export class Multicall {
     private abFlags: Record<string, boolean>
   ) {}
 
-  async call(providerUrls: MulticallProviderUrls, request: MulticallRequestConfig<any>, maxTimeout: number) {
+  async call(
+    providerUrls: MulticallProviderUrls,
+    request: MulticallRequestConfig<any>,
+    maxTimeout: number,
+    isLargeAccount: boolean
```

Comment:
i think Multicall shouldn't know about large or small accounts, probably it's better to use some technical naming here, like shouldUseAlchemy

Reply:
My thought is that it's pretty important to have the isLargeAccount flag for all events across the app, even in the multicall. The reason is that it's the most important cohort for us, and it's better to be able to group any metrics by this flag without name mapping

Reply:
you can add the flat to GlobalMetricData

Reply:
and set it in useConfigureMetrics

Reply:
Yep. This is what i've already done. But as i can see GlobalMetricData works only for events. Not for counter and timings. So i kept this flags for this calls inside multicall.